x/liquidity/spec/04_messages.md

```diff
@@ -1,95 +1,101 @@
+- The balance of `PoolCreator` does not have enough coins for `PoolCreationFee`
 
 ## MsgDepositWithinBatch
 
-Deposit coins to the specified liquidity pool.
+Coins are deposited in a batch to a liquidity pool with the `MsgDepositWithinBatch` message.
 
 ```go
 type MsgDepositWithinBatch struct {
-	DepositorAddress    string         // account address of the origin of this message
-	PoolId              uint64         // id of the liquidity pool where this message is belong to
-	DepositCoins 	    sdk.Coins      // deposit coins of this pool deposit message
+    DepositorAddress    string         // account address of depositor that originated this message
+    PoolId              uint64         // id of the liquidity pool to receive deposit
+    DepositCoins         sdk.Coins      // deposit coins
 }
 ```
 
-**Validity check**
+## Validity checks
 
-- `MsgDepositWithinBatch` failes if
-  - `Depositor` address does not exist
-  - `PoolId` does not exist
-  - if the denoms of `DepositCoins` are not composed of `ReserveCoinDenoms` of the `LiquidityPool` with given `PoolId`
-  - if the balance of `Depositor` does not have enough amount of coins for `DepositCoins`
+The MsgDepositWithinBatch message performs validity checks. The transaction that is triggered with the `MsgDepositWithinBatch` message fails if:
+
+- `Depositor` address does not exist
+- `PoolId` does not exist
+- The denoms of `DepositCoins` are not composed of existing `ReserveCoinDenoms` of the specified `LiquidityPool`
+- The balance of `Depositor` does not have enough coins for `DepositCoins`
 
 ## MsgWithdrawWithinBatch
 
-Withdraw pool coin from the specified liquidity pool.
+Withdraw coins in batch from liquidity pool with the `MsgWithdrawWithinBatch` message.
 
 ```go
 type MsgWithdrawWithinBatch struct {
-	WithdrawerAddress string         // account address of the origin of this message
-	PoolId            uint64         // id of the liquidity pool where this message is belong to
-	PoolCoin          sdk.Coin       // pool coin sent for reserve coin withdraw
+    WithdrawerAddress string         // account address of the origin of this message
+    PoolId            uint64         // id of the liquidity pool to withdraw the coins from
+    PoolCoin          sdk.Coin       // pool coin sent for reserve coin withdrawal
 }
 ```
 
-**Validity check**
+## Validity checks
+
+The MsgWithdrawWithinBatch message performs validity checks. The transaction that is triggered with the `MsgWithdrawWithinBatch` message fails if:
 
-- `MsgWithdrawWithinBatch` failes if
-  - `Withdrawer` address does not exist
-  - `PoolId` does not exist
-  - if the denom of `PoolCoin` are not equal to the `PoolCoinDenom` of the `LiquidityPool` with given `PoolId`
-  - if the balance of `Depositor` does not have enough amount of coins for `PoolCoin`
+- `Withdrawer` address does not exist
+- `PoolId` does not exist
+- The denom of `PoolCoin` are not equal to the `PoolCoinDenom` of the `LiquidityPool`
```

Comment:
see comment and questions on https://github.com/tendermint/liquidity/pull/201/files#r609960982